src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -31,6 +50,18 @@
                            let-node>
                 <span *ngIf="node.data.name"
                       class="me-3">
+                  <span *ngIf="(node.data.type === 'zone' && (node.data.access_key === '' || node.data.access_key === 'null'))">
+                    <i  class="user-warning"
```

Comment:
You don't need to create another class for this!
```suggestion
                    <i  class="text-danger"
```